### DISCUSSION TITLE
log error

### DISCUSSION THREAD

**Discussion by aa1135435558** [Chia-Network/chia-blockchain#1830] [General]
I used a 1T SSD, p two files at the same time, but the second always presents this error message
![5D8FF280-70C2-46AB-9D82-6AC8902DAD90](https://user-images.githubusercontent.com/67729044/114406709-566b9900-9bda-11eb-8158-f6429164a723.jpeg)

**Comment by hoffmang9** [comment]
This is either an error on the storage but is more likely an error with your RAM. It could be that you are running out of RAM or that your RAM is unstable or a stick is bad.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
